stubs/console-menu/consolemenu/menu_formatter.pyi

```diff
@@ -0,0 +1,55 @@
+from consolemenu.console_menu import MenuItem
+from consolemenu.format import MenuBorderStyleType
+from consolemenu.format.menu_borders import MenuBorderStyle as MenuBorderStyle, MenuBorderStyleFactory as MenuBorderStyleFactory
+from consolemenu.format.menu_style import MenuStyle as MenuStyle
+from consolemenu.menu_component import (
+    Dimension as Dimension,
+    MenuFooter as MenuFooter,
+    MenuHeader as MenuHeader,
+    MenuItemsSection as MenuItemsSection,
+    MenuPrompt as MenuPrompt,
+    MenuTextSection as MenuTextSection,
+)
+
+class MenuFormatBuilder:
+    def __init__(self, max_dimension: Dimension | None = ...) -> None: ...
+    def set_border_style(self, border_style: MenuBorderStyle) -> MenuFormatBuilder: ...
```

Comment:
This goes for all these setters:

```suggestion
    def set_border_style(self: Self, border_style: MenuBorderStyle) -> Self: ...
```

where Self is `from _typeshed import Self`